src/map/homunculus.cpp

```diff
@@ -310,7 +310,7 @@ int hom_vaporize(map_session_data *sd, int flag)
 * @param hd
```

Comment:
Comment above says "send negative to disable".
This does not work anymore.
Please remove this part of the comment.

Reply:
![image](https://github.com/user-attachments/assets/2d4b278d-370f-4c02-8d9d-a6cbf973574b)
i changed to int8 because this function send -1

Reply:
if this is wrong i will change it again